### PR TITLE
Rspec testing

### PR DESCRIPTION
**Pull request by alissonperes**
implemented testing with rspec and fixed minor bugs found while testing.

### CODE REVIEW

File: spec/game_spec.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      expect(game.board).to be_kind_of Hash
+    end
+    it 'game_over starts as nil' do
+      expect(game.game_over).to be_nil
+    end
+    it 'winner starts as nil' do
+      expect(game.winner).to be_nil
+    end
+  end
+end
```
Comment (Redvanisation) [inline spec/game_spec.rb]: These tests are covering only the positive cases, please write tests for the negative cases as well.

File: spec/game_spec.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    expect(game.check_win('O')).to be false
+  end
+end
+
+describe Game, '#possible_wins' do
+  let(:game) { Game.new }
+  it '@possible is an array' do
+    expect(game.possible).to be_kind_of Array
+  end
+end
```
Comment (Redvanisation) [inline spec/game_spec.rb]: Please write tests for the negative cases here as well.
